### ISSUE TITLE
Bug: NVDA is unable to announce the last letter when using the review cursor in a part of Windows Settings

### ISSUE THREAD

**Issue by ElCrackLoko** [nvaccess/nvda#13837]
### Steps to reproduce:
1. Open the Settings App, Windows+I. And search About Device or About, in Windows 10 and in Windows 11, Information.
2. Replicated in Windows 11, but surely in Windows 10 it happens too.
2.1. Within the info tab, look for the "show all settings" buttons and check that these are expanded for either the Windows Specifications or Device Specifications groupings. Try to make use of the review cursor within the groupings, I repeat, within the groupings, not on the groupings. IF it is done on within the groupings, the problem will not be replicated.
2.2. Try reading your username, your edition of Windows, or the full name of your processor.
### Actual behavior:
NVDA is not able to get to the last letter. For example, Processor, the user will only be able to read up to O without being able to get to R with the review cursor.
### Expected behavior:
It is expected that through the review cursor it can be read normally, for example, Username
#### NVDA installed/portable/running from source:
NVDA complete and properly installed.

#### NVDA version:
2022.2.0 Beta2

#### Windows version:
WIndows 11 22H2 (x64) build 22621.160

#### Name and version of other software in use when reproducing the issue:
None.

#### Other information about your system:
Windows 11, Insider.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Same behaviors.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes that's how it is.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes, that's how it is.

**Comment by Adriani90** [comment]
Closing as duplicate of #13302.
